notebooks/binary_predicates.ipynb

```diff
@@ -0,0 +1,1157 @@
+{
```

Comment:
<blockquote>There are only two relationships that a point can have with another point: equality or inequality.</blockquote><p> </p><p>I understand what you mean here. Although for those who don't have much context in binpreds, point-point can also be disjoint, or intersects. But they can all be reduced to equal/inequality. Perhaps rephrase the word here for general audience?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cuspatial/pull/1156/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>I gave it another pass.</p>